Quaver.API/Maps/Structures/SliderVelocityInfo.cs

```diff
@@ -72,6 +72,19 @@ public void SetMultiplier(float multiplier)
             Multiplier = multiplier;
         }
 
+        /// <inheritdoc />
+        public int CompareTo(SliderVelocityInfo other)
+        {
+            if (ReferenceEquals(this, other))
+                return 0;
+
+            if (other is null)
+                return 1;
+
+            var compare = StartTime.CompareTo(other.StartTime);
+            return compare is 0 ? Multiplier.CompareTo(other.Multiplier) : compare;
+        }
+
```

Comment:
This change in sorting behavior potentially breaks certain SV effects in already existing SV maps if they have overlapping SVs (of which many ranked SV maps do have).

To a much lesser extent this is also a potential problem for the other object types.

Reply:
That's true, do I simply return -1 if `StartTime` is equal? I assume `this` is always the first argument, and therefore should be considered lesser.

Reply:
Nevermind, `List<T>.Sort` does not uphold this guarantee. I'll work on fixing this in a moment.